docs/src/DeveloperDocumentation/styleguide.md

```diff
@@ -190,6 +190,54 @@ end
 However, as always, rules sometimes should be broken.
 
 
+## Optional arguments for parents of return values
+
+Several objects in OSCAR have `parent`s, e.g. polynomials, group elements, ... 
```

Comment:
You could also link to the relevant section of the AbstractAlgebra documentation here?

Reply:
No clue how to do that.